addon/mixins/node-actions.js

```diff
@@ -0,0 +1,100 @@
+import Ember from 'ember';
+
+export default Ember.Mixin.create({
+    node: null,
+    model: null,
+    _node: Ember.computed.or('node', 'model'),
```

Comment:
I just don't understand where 'node' is getting set in the first place.   On the node detail route, I see the `return this.store.findRecord('node', params.node_id);` - because it is looking up an individual node, is that how it knows what this.get('node') is without having to explicitly define 'node'?